### DISCUSSION TITLE
Short option for --target

### DISCUSSION THREAD

**Discussion by bessarabov** [umputun/spot#48] [General]
Here is the fragment from `spot --help`:

```
-t, --task=       task name
...
-d, --target=     target name (default: default)
```

1. I'm very curious — why did you choose `-d` for short option name `--target`. What does `-d` means?

2. For me it is a pity that the short name for `--target` is not `-t`. I understand that this is because `-t` is used for `--task`. But does to `--task` really need the short option? I think that the target is more important than the task, so I would prefer to use `-t` for `--target`.

**Comment by umputun** [comment]
Well, -t already is in use for task name. by -d I meant "destination"

**Comment by bessarabov** [comment]
Yes, thank you. Now it is clear what `-d` means.

I think that the target is more important than the task (I think that I more ofter will specify target in the command line than the task), so what do you think to use `-t` as a short option for `--target` and use some other short command for `--task`?

**Comment by bessarabov** [comment]
Oh! I see that you have made `-t` the short option for `--target` https://github.com/umputun/spot/commit/2acd38b22314b0ac34ac0fef437c50b6ad7d1f04

Thank you! I'm closing this discussion.
